definitions/features/pulp3.rb

```diff
@@ -0,0 +1,22 @@
+require 'foreman_maintain/utils/service/systemd'
+
+class Features::Pulp3 < ForemanMaintain::Feature
+  metadata do
+    label :pulp3
+
+    confine do
+      ForemanMaintain::Utils::Service::Systemd.new('pulpcore-api', 0).exist?
+    end
+  end
+
+  def services
+    [
+      system_service('pulpcore-api', 10),
+      system_service('pulpcore-content', 10),
+      system_service('pulpcore-resource-manager', 10),
+      system_service('pulpcore-worker@*', 20, :all => true),
```

Comment:
I tried a  'foreman-maintain service restart'  and it didn't seem to properly start the pulpcore-worker services, did it work for you?

Reply:
I see this issue as well

Reply:
looks like the `:all => true` isn't making it to the actual `systemctl` command.